droidlet/interpreter/craftassist/tasks.py

```diff
@@ -961,35 +961,52 @@ class Get(Task):
 
     Examples::
 
-        >>> task_data = {"idm": (319, 0), "pos": (1, 0, 1), "eid": 11, "memid": 99}
+        >>> task_data = {"pos": (1, 0, 1), "eid": 11, "memid": 99}
         >>> dance = Dance(agent, task_data)
     """
 
     def __init__(self, agent, task_data):
         super().__init__(agent)
-        self.idm = task_data.get("idm")
         self.pos = task_data["pos"]
         self.eid = task_data["eid"]
         self.obj_memid = task_data["obj_memid"]
         self.approx = 1
-        self.attempts = 10
-        if self.idm is not None:
+        self.attempts = 20
+        self.type_name = agent.memory.get_mem_by_id(self.obj_memid).type_name
```

Comment:
what does the type_name field represent ?

Reply:
the name of the object being dropped, e.g. "stone".